### PR TITLE
Determining device form, idiom and system version

### PR DESCRIPTION
**Pull request by mathieuwilsonnv**
Squashed commit:

[4301ace9] Made separate files for AnalyticsVersionInfo, iOS and Android

[bd27225c] Removed AnalyticsInfo and AnalyticsVersionInfo from the generated folder.

Github issue: #74 

## PR Type
What kind of change does this PR introduce?
Feature

## What is the new behavior?
It is now possible to determine the device form, idiom and system version by using the AnalyticsInfo and AnalyticsVersionInfo classes on UWP, Android and iOS.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x ] Tested code with current [supported SDKs](../README.md#supported)
- [x ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [x ] Contains **NO** breaking changes

Internal issue :
https://nventive.visualstudio.com/Umbrella/Umbrella%20Team/_workitems/edit/123761

### CODE REVIEW

File: doc/ReleaseNotes/_ReleaseNotes.md [hunk 1]
```diff
@@ -15,10 +16,4 @@
  * 134026 [iOS] Setting a different DP from TextBox.TextChanging can cause an infinite 'ping pong' of changing Text values
  * 134415 [iOS] MenuFlyout was not loaded correctly, causing templates containing a MenuFlyout to fail
  * 133247 [iOS] Image performance improvements
- * 135192 [iOS] Fixed ImageBrush flash/flickering occurs when transitioning to a new page.
  * 135112 [Android] Fix crash in UpdateItemsPanelRoot() in the ItemsControl class.
- * 132014, 134103 [Android] Set the leading edge considering header can push groups out off the screen
- * 131998 [Android] Window bounds set too late
- * 131768 [iOS] Improve ListView.ScrollIntoView() when ItemTemplateSelector is set
```
Comment (MatFillion) [inline doc/ReleaseNotes/_ReleaseNotes.md]: this is not normal, you shouldn't delete lines here.

File: doc/ReleaseNotes/_ReleaseNotes.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+*123761 - Added AnalyticsInfo to get the device type (idiom) and the device system.
```
Comment (jeromelaban) [inline doc/ReleaseNotes/_ReleaseNotes.md]:
```suggestion
* 123761 - Added AnalyticsInfo to get the device type (idiom) and the device system.
```

File: src/Uno.UWP/System/Profile/AnalyticsVersionInfo.Android.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+			{
+				string idiom = "";
+
+				try
+				{
+					// Threshold in inches for diagonal screensize between what HardwareHelper will consider a tablet or a phone.
+					//This is the value given to us by microsoft documentation : https://docs.microsoft.com/en-us/windows/uwp/design/devices/
+					double _tabletMinimumSizeThreshold = 7.0;
+					
+					var displayMetrics = Application.Context.Resources.DisplayMetrics;
```
Comment (jeromelaban) [inline src/Uno.UWP/System/Profile/AnalyticsVersionInfo.Android.cs]: Application.Context may be null, if called to early.

Comment (MatFillion) [reply src/Uno.UWP/System/Profile/AnalyticsVersionInfo.Android.cs]: @jeromelaban what adjustment would you suggest? We could check if it's null and throw InvalidOperationException saying we're probably calling this too fast?

**Comment by ghuntley** [comment]
Superseded by https://github.com/nventive/Uno/pull/1110

**Comment by ghuntley** [comment]
@all-contributors please add @mathieuwilsonnv for his code contributions

**Pull request closed without merging**